src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('h1');
+
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+promise1.then((resolvedText) => makeANotification('message', resolvedText));
+
+promise2.catch((errorText) => makeANotification('message error-message',
+  errorText));
+
+function makeANotification(classType, text) {
```

Comment:
It is a good tone to define a function before use. 
Referring to a function before declaration harms readability and maintainability.
Please move it to the top